Automattic/wp-calypso
# Seller Experience - Stepper: Update products-list resolver to use thunks instead of generators

#### Changes proposed in this Pull Request

Update products-list resolver to use thunks instead of generators

#### Testing instructions

- Check out this branch
- Run `yarn test-packages packages/data-stores/src/products-list/test/`

Related to #63246
